gax/src/main/java/com/google/api/gax/rpc/Callables.java

```diff
@@ -102,6 +105,23 @@ private Callables() {}
         innerCallable, retryingExecutor, callSettings.getResumptionStrategy());
   }
 
+  /*
+   * Returns the default Duration for a single RPC call given a Callable's RetrySettings. This
+   * configuration most likely does not belong in retry settings and may change in the future.
+   */
+  static Duration singleRpcCallTimeout(RetrySettings retrySettings) {
+    // Prefer initialRpcTimeout, then maxRpcTimeout, then totalTimeout
```

Comment:
why not the minimum of the 3?

Reply:
We could, although by the intended definitions of these settings, initialRpcTimeout should be smaller than maxRpcTimeout which should be smaller than totalTimeout.

Reply:
fair point